web/static/js/consent.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (function() {
+  var noConsent = document.querySelectorAll(".no-consent");
+  var yesConsent = document.querySelectorAll(".yes-consent");
+
+  noConsent.forEach(function(el){
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach - browser support isn't there for IE

Maybe use: `[].forEach.call(noConsent, function (el)...` instead, what's the browser compatibility spec for this?